holmes.go

```diff
@@ -667,3 +700,26 @@ func (h *Holmes) Set(opts ...Option) error {
 	}
 	return nil
 }
+
+func (h *Holmes) DisableProfileReporter() {
+	atomic.StoreInt32(&h.opts.pReportOpts.active, 0)
+	h.opts.pReportOpts.cancelCh <- struct{}{}
+}
+
+func (h *Holmes) EnableProfileReporter() (err error) {
```

Comment:
I think it's good enough to omit this method.

Also, we do not start the consumer goroutine here after `DisableProfileReporter`, it's a bug.

Reply:
No allow user enable it and disable it then enable it again?